cmd/manager/main.go

```diff
@@ -67,6 +63,35 @@ func main() {
 	// uniform and structured logs.
 	logf.SetLogger(zap.Logger())
 
+	viper.BindPFlags(pflag.CommandLine)
```

Comment:
Another thing, I also added a "reap" value into a ReconcilerConfig that would take the REAPER environmental variable and instantiate the reconciler with that value. You may want to incorporate that value into this setup when you merge